egs2/itako/svs1/local/prep_segments_from_xml.py

```diff
@@ -99,20 +99,21 @@ def make_segment(file_id, tempo, notes, threshold, sil=["P", "B"]):
                 segment = SegInfo()
         # remove rest note
         if "itako04" in file_id:
-            if i in [191, 193, 194, 195, 197, 199, 200, 201, 202, 203, 207]:
```

Comment:
Given their future potential data updates, how about we explicitly note the version of the dataset (if no specific version number if provided, let's simply use date) in README for user's reference?